pkg/store/project_head/interface.go

```diff
@@ -0,0 +1,7 @@
+package projecthead
```

Comment:
the folder name should be `pkg/store/projecthead

Reply:
updated